Modules/TOF/src/TaskRaw.cxx

```diff
@@ -47,9 +47,23 @@ void RawDataDecoder::rdhHandler(const o2::header::RAWDataHeader* rdh)
 {
   mCounterRDH[rdh->feeId & 0xFF].Count(0);
 
+  //Fatal
```

Comment:
Maybe here it is worth to extend a bit the comment, fatal is a bit too harsh maybe and we can consider printing warning messages, what do you think?

Reply:
Ciao @njacazio, many thanks for the comments!
So, fatal is how Roberto refers to it in https://github.com/AliceO2Group/AliceO2/pull/5802/commits/90c8e48b4701eb30e5c0aa90acc517e983e112c3 .
How would you suggest to change it?

For the warning: sounds great, where you thinking of a LOG(WARNING) or using a checker?

Reply:
I mean, I don't think we want to throw a fatal error if we encounter this word here, therefore I would at least call it `Case for the RDH word: fatal` or something that to explain that it is not an error in our task.
I guess you can put the LOG and we will implement the checker as well to inform the QC crew.
Does it seem reasonable?

Reply:
Everything clear, sounds perfect! I'm on it